tools/keepalived/keepalived/check/ipvswrapper.c

```diff
@@ -300,18 +300,19 @@ ipvs_group_range_cmd(int cmd, ipvs_service_t *srule, ipvs_dest_t *drule, virtual
 static bool
 is_vsge_alive(virtual_server_group_entry_t *vsge, virtual_server_t *vs)
 {
+	unsigned t = -1;
```

Comment:
It's a little strange to assign -1 to an unsigned variable.

Reply:
yes，do you have any suggestion on overflow judgement